projects/cashmere/src/lib/scroll-nav/nav/scroll-nav.component.ts

```diff
@@ -36,5 +36,6 @@ export class HcScrollNavComponent implements AfterViewInit {
             e.classList.remove(this.ACTIVE_CLASS);
         });
         element.classList.add(this.ACTIVE_CLASS);
+        element.focus();
```

Comment:
As I think about this more, changing the focus on scroll makes me a little nervous. Could make for an odd UX in some cases. Like if the user was filling out form or interacting with controls in the content area, and then scrolled, I think the browser would steal focus away from the form and place it on the scroll nav.

Could we instead just make the focus outline less noticeable, like a light dashed or dotted line or something?

Reply:
Or maybe even just call blur on the nav element when you scroll and the next or prev section needs to be activated.

Reply:
I think calling blur on scroll would work for me.  I do think it's odd to keep the clicked item highlighted on scroll, and blur would resolve that.  And I'd guess that calling blur on the nav element wouldn't remove focus from a form element in the example you gave.

So I'd be good with that change if you'd feel better about that approach.

Reply:
Made that update @corykon - I like this change.

Reply:
Sweet yeah UX wise I that's prob safer. I'd recommend something like this for calling blur, so we're only firing off events when we need to.

```
private setActiveClass(element: HTMLElement): void {
        const focusedEl = document.activeElement;
        this._links.forEach(e => {
            e.classList.remove(this.ACTIVE_CLASS);
            if ( focusedEl === e) { e.blur(); }
        });
        element.classList.add(this.ACTIVE_CLASS);
    }
```

Reply:
Even better!  Thanks @corykon 👍